src/WebServer/Http/HttpServer.cs

```diff
@@ -19,9 +20,11 @@ public class HttpServer : IDisposable
         private readonly StreamSocketListener _listener;
         private readonly SortedSet<RouteRegistration> _routes;
         private readonly ContentEncoderFactory _contentEncoderFactory;
+        private ILogger _log;
 
         public HttpServer(int serverPort)
         {
+            _log = LogManager.GetLogger<HttpServer>();

```

Comment:
What's the plan for setting the Log level? I think we should support this very basic feature.